app/views/comments/_comment_form.html.erb

```diff
@@ -28,19 +28,19 @@
 
       <% if comments_are_moderated(commentable) && !current_user_is_work_creator(commentable) %>
         <p class="notice">
-          <%= ts("This work's creator has chosen to moderate comments on the work. Your comment will not appear until it has been approved by the creator.") %>
+          <%= t(".moderation_forewarning") %>
         </p>
       <% end %>
 
       <% if logged_in? %>
         <% if current_user_is_anonymous_creator(commentable) %>
           <p class="notice">
-           <%= ts("While this work is anonymous, comments you post will also be listed anonymously.") %>
+          <%= t(".anonymous_forewarning") %>
           </p>
         <% end %>
 
         <% if current_user.pseuds.count > 1 %>
-          <h4 class="heading"><%= ts("Comment as") %> <%= f.collection_select :pseud_id, current_user.pseuds, :id, :name, {:selected => (comment.pseud ? comment.pseud.id.to_s : current_user.default_pseud.id.to_s)}, :id => "comment_pseud_id_for_#{commentable.id}", :title => ts("Choose Name") %>
+          <h4 class="heading"><%= t(".comment_as") %> <%= f.collection_select :pseud_id, current_user.pseuds, :id, :name, { selected: (comment.pseud ? comment.pseud.id.to_s : current_user.default_pseud.id.to_s) }, id: "comment_pseud_id_for_#{commentable.id}", title: t(".choose_name") %>
```

Comment:
There is the `.comment_field_title` key, but this field title doesn't have `field_title` in its key. So maybe `.choose_name_field_title` is better? Not sure though.